addons/api/mirage/route-handlers/auth.js

```diff
@@ -4,7 +4,6 @@
  */
 
 // import config from '../../config/environment';
```

Comment:
I think we can remove this comment?

Reply:
I didn't want to delete this comment or the rest of commented stuff within this file since is out of the scope of migrating `ember-uuid` to `uuid`.
`config` is used within `authenticateHandler` (which is commented) and I do not know why is commented or even the scope of it.

I will create a ticket to clean up `authenticateHandler`.

Reply:
[Here link to ticket](https://hashicorp.atlassian.net/browse/ICU-15896)